src/app/page.tsx

```diff
@@ -113,9 +114,28 @@ const Communicate: FC<{}> = () => {
     return <Button onClick={() => showCommunicationNotification()}>Contact Researcher</Button>
 }
 
-const Approve: FC<{ fileName: string }> = () => {
+const Approve: FC<{ fileName: string }> = ({fileName}) => {
+    const router = useRouter()
```

Comment:
My nextjs knowledge is too minimal to immediately know why, but when there are results for approval I currently get an error here in the UI :man_shrugging: . Perhaps it's something wrong with my environment, but I figured I'd note just in case.

Reply:
interesting - what is the error? we haven't hit this yet

Reply:
Here's what I see for whatever reason:
![image](https://github.com/user-attachments/assets/c2424567-4505-407d-8781-97be562700e3)

Reply:
weird - not able to reproduce this - if you try from a clean slate do you still get it?